web/src/pages/account-setup.tsx

```diff
@@ -112,8 +111,7 @@ const AccountSetupPage = (): ReactElement => {
   };
 
   return (
-    <PageSkeleton>
-      <NavBar logoOnly="NAVIGATOR_MYNJ_LOGO" />
+    <PageSkeleton showNavBar logoOnly="NAVIGATOR_MYNJ_LOGO">
```

Comment:
Might get some pushback on using `showNavBar` in place of `showNavBar={true}` - right now the code style doc refers to it this way but I'm open to changing it to use the other style

Reply:
No pushback from me! This is fine, until we decide a common strategy and institutionalize it via ESLint.